downstream/modules/platform/proc-upgrade-installer.adoc

```diff
@@ -33,7 +33,12 @@ The setup script pauses and indicates that a "pre-2.x" inventory file was detect
 NOTE: By running the setup script, the Installer modified a few fields from your original inventory file, such as renaming [tower] to [automationcontroller].
 . Modify the newly generated `inventory.new.ini` file to configure your automation mesh by assigning relevant variables, nodes, and relevant node-to-node peer connections:
 +
-NOTE: The design of your automation mesh topology depends on the automation needs of your environment. The example below offers one possible scenario for automation mesh design.
+[NOTE]
+====
+The design of your {AutomationMesh} topology depends on the automation needs of your environment.
+It is beyond the scope of this document to provide designs for all possible scenarios.
+The following is one example {AutomationMesh} design.
```

Comment:
For admonitions that include more than one line of text I think the convention is to use the format:

```
[ADMONITION_TYPE]
====
Lorem ipsum
====
```

Reply:
I'm also a bit of a stickler for keeping each sentence on a separate line. Asciidoc automatically forms paragraphs from consecutive lines of text. Putting each sentence on a separate line makes it easier to review files.